### ISSUE TITLE
Pipelines

### ISSUE THREAD

**Issue by mrcomac** [scrapy-plugins/scrapy-splash#87]
Hi!

I have a spider with default scrapy.request working fine. after each item the pipeline is called. But, if i change to SplashRequest, the pipeline function wasn't called.

**Comment by Gallaecio** [comment]
@marcosmachado81 It’s been a long time. Did you ever manage to solve this issue?

**Comment by evanjs** [comment]
I am also experiencing this issue.
I found #92 in trying to solve my initial recursion issue, and am not sure how valid my mix of the proposed solutions is (it does follow and crawl recursively, so... :man_shrugging:)

Regardless, results do not seem to propagate to my pipeline, though my run log does indicate said pipeline is enabled.

**Comment by evanjs** [comment]
Okay, so in my case, I was getting some debug messages, which indicated that `strip()` isn't a valid member of list, etc etc, whatever.  I saw `DEBUG` and just kept moving.

As far as I'm concerned, the spider was dying and never returning.
I fixed the few instances of strip on lists, etc, and now everything seems to work fine.

Though, I still have no clue how relevant this is to OP's case.

On a related note... it might've been helpful if the DEBUG messages were ERROR or similar.  Not sure what's best, but I'm sure I would've been less likely to ignore them.
